baselines/defender.md

```diff
@@ -649,20 +738,28 @@ learning and analysis techniques to detect malicious intent. Enabling
 this feature may slow down message delivery to the user due to the
 scanning.
 
-### 2.8.1 Policy
+### Policies
```

Comment:
Since Policy is a section header should it be consistent through the document as Policy or Policies?